CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contribution Guidelines
+
+Hello. 👋
+
+Welcome to the newest project for FreeCodeCamp. We are glad you are here!
+
+We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It's only 196 words long.
+
+Happy contributing.
+
+## We are just getting started
+
+Implementation details are still being discussed and the project is currently at a very early stage.  The stage of the project is changing on a daily basis, thoughts and ideas are being contributed at a very high pace.
+
+## What do we need help with right now?
+
+We are in the very early stages of development on this new application. We value your insight and expertise.  In order to prevent duplicate issues, please search through our existing issues to see if there is one for which you would like to provide feedback. We are currently trying to consolidate many of the issues based on topics (Docs, UI/UX, API architecture, Tech Stack, etc)
+
+## How to submit an issue
+
+- Search through current issues to verify that you are not creating a duplicate issue.
+- Use the issue template as a guide to tell us more about your issue:
+  - **Is your feature request related to a problem? Please describe.**
+      A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+  - **Describe the solution you'd like**
+      A clear and concise description of what you want to happen.
+
+  - **Describe alternatives you've considered**
+      A clear and concise description of any alternative solutions or features you've considered.
+
+  - **Additional context**
+      Add any other context or screenshots about the feature request here.
+
+- Be as descriptive as possible.  At this early stage, it is important to have qualitative evidence to support or dissent on an issue.
+
+## Working on a change
+
+- Create a new branch on your fork: `git checkout -b your-branch-name`
+- Use the Github editor or your local environment to make your changes
+- Push your changes to the branch you created on your fork: `git push origin your-branch-name`
+
+## How to submit a pull request
+
+- After working on changes and pushing to your fork of the repo, create a pull request on your fork.
+- Complete the PR template:
```

Comment:
Again, [GitHub's pull request templates](https://help.github.com/en/articles/creating-a-pull-request-template-for-your-repository) would be useful for this.